.gitattributes

```diff
@@ -0,0 +1,2 @@
+# Ensure LF line endings across platforms, especially when comparing test fixtures
+*.js text eol=lf
```

Comment:
Do we need a newline at EOF here?

Reply:
Yes, good catch :)

Reply:
Ah, it's already there, actually. GitHub has an icon if it's missing.